packages/shopping-cart/src/types.ts

```diff
@@ -363,6 +364,13 @@ export interface GSuiteProductUser {
 	password: string;
 }
 
+export interface TitanProductUser {
+	alternative_email: string;
+	email: string;
+	name: string;
+	password: string;
+}
```

Comment:
Given that this prop could be specified from the client or returned from the server, we actually need the following updates:
1. `alternative_email` and `name` are optional
2. We need both `encrypted_password` and `password`, with both needing to be optional
3. We should add `is_admin` as an optional boolean
```suggestion
export interface TitanProductUser {
	alternative_email?: string;
	email: string;
	encrypted_password?: string;
	is_admin?: boolean;
	name?: string;
	password?: string;
}
```